### DISCUSSION TITLE
Command Search : assigning hotkeys - Ctrl+Enter starts the process (like Ctrl+double-click does)

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#1266] [Ideas]
Maybe this is well-know but I did not know it.

In Command Search, with a menu item selected, pressing Ctrl+Enter invokes the hotkey mapping dialog (as does Ctrl+Shift+Enter and Ctrl+Alt+Enter).  That's in addition to Ctrl+double-click documented on the website.

As a non-mouseketeer, I appreciate this feature.  I'm not always good at double-clicking.

Please let me know if I have this goobered.  I plan to soon post a youtube video supplementing one I already posted on Command Search.  Thanks.

**Comment by macmarrum** [comment]
A good point :+1: 
I've updated the docs: [Hot Keys and Beyond](https://macmarrum.github.io/freeplane-docs/user-documentation/hot-keys-and-beyond.html#hot-keys-for-actions)
